client/components/common/CustomDropDown.js

```diff
@@ -14,11 +14,19 @@ class CustomDropDown extends Component {
     super(props);
     this.state = {
       listOpen: false,
-      headerTitle: this.props.title,
+      headerTitle: '',
```

Comment:
```suggestion
      headerTitle: this.props.title
```
This should be left as is, due to the fact that the dropdown still needs an initial text. Which is supplied to it by the parent component.

For more context, the "filters" section on the events page which also uses this component has no initial text is rendered due to this.

Kindly help look into that, thanks

Reply:
okay